pkg/apiserver/endpoints/filters/requester.go

```diff
@@ -64,6 +68,11 @@ func WithRequester(handler http.Handler) http.Handler {
 				}
 			}
 
+			requester, err = signStaticRequester(ctx, idService, requester.(*identity.StaticRequester))
```

Comment:
🤔  maybe we can add the signing to the grpc interceptor 🤔

Reply:
✅